core/arch/riscv64/riscv64.asm

```diff
@@ -226,8 +261,36 @@ GLOBAL_LABEL(dr_setjmp:)
  */
         DECLARE_FUNC(dr_longjmp)
 GLOBAL_LABEL(dr_longjmp:)
-/* FIXME i#3544: Not implemented */
-        ret
+        sd       x18, 0 (ARG1)
```

Comment:
But longjmp should be loading the values from the given dr_jmp_buf into registers. See the equivalent version in core/arch/aarch64/aarch64.asm.

Reply:
Done. Fixed https://github.com/DynamoRIO/dynamorio/pull/6071/commits/3859e2075255495463f4d09652e54771306f0a75